src/components/smart_objects/src/smart_object.cc

```diff
@@ -734,16 +734,28 @@ void SmartObject::duplicate(const SmartObject& OtherObject) {
 void SmartObject::cleanup_data() {
   switch (m_type) {
     case SmartType_String:
-      delete m_data.str_value;
+      if (m_data.str_value) {
+        delete m_data.str_value;
+        m_data.str_value = NULL;
```

Comment:
@SNiukalov Use `nullptr` everywhere instead of `NULL`

Reply:
Fixed in:https://github.com/smartdevicelink/sdl_core/pull/3146/commits/6915e49d17c1a2bb31e654ab1fa2fc3b668b9040